azure-function-app/src/_providers.tf

```diff
@@ -24,7 +24,11 @@ provider "mdxc" {
 }
 
 provider "azurerm" {
-  features {}
+  features {
```

Comment:
I'm a little wary of adding this to a template without a heavy warning. What resources are leftover after an app function decomission?

Reply:
Heavy warning => comment above about how this will delete all yo childen.